libs/api/domains/auth-admin/src/lib/tenant/tenants.service.ts

```diff
@@ -147,4 +147,46 @@ export class TenantsService {
       pageInfo: { hasNextPage: false },
     }
   }
+
+  async getTenantById(id: string, user: User): Promise<Tenant> {
```

Comment:
This function should replace `getTenant` above, so just remove `getTenant` and use this instead.